runtime/interface.go

```diff
@@ -131,6 +131,4 @@ type Metrics interface {
 	ProgramParsed(location common.Location, duration time.Duration)
 	ProgramChecked(location common.Location, duration time.Duration)
 	ProgramInterpreted(location common.Location, duration time.Duration)
-	ValueEncoded(duration time.Duration)
-	ValueDecoded(duration time.Duration)
```

Comment:
These functions are not used anymore: encoding and decoding is done inside of the atree library. We might want to bring these metric callbacks back at some point